src/components/App/App.js

```diff
@@ -2,12 +2,35 @@ import './App.css';
 import banner from '../../images/old-school-theater-banner.png';
 import movieData from '../../data/data';
 import MoviesBox from '../MoviesBox/MoviesBox';
+import SingleMovie from '../SingleMovie/SingleMovie';
+import ExitMovie from '../ExitMovie/ExitMovie';
+import { useState } from 'react';
 
 function App() {
+  const [allMovies, setAllMovies] = useState(movieData.movies);
+  const [singleMovie, setSingleMovie] = useState(null);
+
+  const viewMovie = (id) => {
+    const selectedMovie = allMovies.find(movie => movie.id === id);
+    setSingleMovie(selectedMovie);
+  };
+
+  const viewAll = () => {
+    setSingleMovie(null);
+  };
+
   return (
     <main className="main-app">
-      <img className='main-title' src={banner} alt='old school theater banner with the title rancid tomatillos' />
-      <MoviesBox movies={movieData.movies} />
+      {singleMovie ? 
+        <>
+          <ExitMovie viewAll={viewAll} />
+          <SingleMovie selectedMovie={singleMovie} /> 
+        </> : 
+        <>
+          <img className='main-title' src={banner} alt='old school theater banner with the title rancid tomatillos' />
+          <MoviesBox movies={allMovies} viewMovie={viewMovie}/>
+        </>
```

Comment:
How would you feel if we imported the exit banner image in app.js and did this instead of having a separate component for exit banner since it is so similar to the regular banner (the following code would replace what you currently have on the highlighted lines)
```
      <img className='main-title' src={singleMovie ? exitBanner : banner} onClick={viewAll} alt='old school theater banner with the title rancid tomatillos' />
      {singleMovie ? <SingleMovie selectedMovie={singleMovie} /> : <MoviesBox movies={allMovies} viewMovie={viewMovie}/>}
```

Let me know your thoughts, I just felt like this was a bit cleaner

Reply:
I was planning on changing this this weekend to have it be to the back button only. That way the banner will always be in the App component and this component will hold the back button that gets turned on and off depending on what page you're on. I really don't like how the text moves when you click on a cover.

Reply:
oh!! I see! Sounds good to me